pulpcore/cli/ostree/repository.py

```diff
@@ -21,6 +22,7 @@
     label_select_option,
     list_command,
     name_option,
+    pulp_option,
```

Comment:
Can you make sure, that the required pulp-cli version (0.12) contains `pulp_option` with the `needs_plugin`?

Reply:
The argument was added in version 0.13. I am bumping the requirement.